### PR TITLE
Translated the code to English and corrected for PEP8

### PR DESCRIPTION
**Pull request by Franco98404**

### CODE REVIEW

File: MACHINE_LEARNING/machine_learning/models/main.py [hunk 1]
```diff
@@ -9,17 +9,13 @@
 # accordance with the terms of the license agreement you entered into
 # with Jalasoft.
 #
-from prediccion import Prediccion
+from prediction import Prediction
 
-# noinspection SpellCheckingInspection
 if __name__ == '__main__':
-    # Definir el modelo de detección de objetos ("nasnet", "vgg", "resnet")
```
Comment (awielern) [inline MACHINE_LEARNING/machine_learning/models/main.py:16]: Please translate the comments into english

**Review by awielern — COMMENTED** [review]
Please translate the comments into english

File: MACHINE_LEARNING/machine_learning/models/main.py (hunk 1 above)
Comment (Franco98404) [reply MACHINE_LEARNING/machine_learning/models/main.py:16]: That line was removed. So is not necessary to translate that line.

File: MACHINE_LEARNING/machine_learning/models/resnet.py [hunk 2]
```diff
@@ -16,15 +16,13 @@
 import numpy as np
 
 
-# noinspection SpellCheckingInspection
 class ResNet:
     """Modelo ResNet50"""
     def __init__(self):
         # Definir el modelo preentrenado
```
Comment (MatiasBonifacio) [inline MACHINE_LEARNING/machine_learning/models/resnet.py]: Please translate the comments

File: MACHINE_LEARNING/machine_learning/models/prediction.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#
+# @resnet.py Copyright (c) 2021 Jalasoft.
```
Comment (MatiasBonifacio) [inline MACHINE_LEARNING/machine_learning/models/prediction.py]: Should be @prediction.py

**Review by MatiasBonifacio — APPROVED** [review]
For the next commit please translate the comments

File: MACHINE_LEARNING/machine_learning/models/resnet.py [hunk 4]
```diff
@@ -16,15 +16,13 @@
...
         self.name = 'ResNet50'
         self.model = ResNet50(weights='imagenet', include_top=True)
 
-    # noinspection SpellCheckingInspection
     # Método que realiza la predicción de objetos en una imagen
```
Comment (awielern) [inline MACHINE_LEARNING/machine_learning/models/resnet.py]: Please translate to english from here to the end

**Review by awielern — COMMENTED** [review]
Please complete the translation
In the yolo.py file, there is a variable "indices", it should be in english

File: MACHINE_LEARNING/machine_learning/models/prediction.py (hunk 3 above)
Comment (Franco98404) [reply MACHINE_LEARNING/machine_learning/models/prediction.py]: You are right! Thank you!

File: MACHINE_LEARNING/machine_learning/models/resnet.py (hunk 4 above)
Comment (Franco98404) [reply MACHINE_LEARNING/machine_learning/models/resnet.py]: Solved! Thank you!

**Comment by Franco98404** [comment]
> Please complete the translation In the yolo.py file, there is a variable "indices", it should be in English - @awielern 

I thought that you are right but the video of implementation of this model uses "indices".  Check this link:
https://www.youtube.com/watch?v=Sp9mEGubBJs&list=PLMoSUbG1Q_r8nz4C5Yvd17KaXy8p0ufPH&t=165s

That was more confusing to me, so I decided to research more about it and found this article: 
https://www.nasdaq.com/articles/indexes-or-indices-whats-the-deal-2016-05-12

So, I think that "indices" is correct, what do you think?

**Review by awielern — APPROVED** [review]
Great job! Everything is already translated into English.

**Review by ChristianTC — APPROVED** [review]
looks good

**Pull request merged**